File: php-phpunit/test/Session1_GreeterTest.php
<?php

use \Wordcount\Greeter;

/**
 * Session 1: GreeterTest - Your first tests.
 */
class Session1_GreeterTest extends \PHPUnit_Framework_TestCase {

    // TODO We will add the proper self.assertions together.

    /** @test */
    function shouldReturnHelloName() {
        $greeter = new Greeter();
        // TODO Check that "Hello Peter", $greeter->greet("Peter").
    }

    /** @test */
    function shouldReturnHelloForNull() {
        $greeter = new Greeter();
        // TODO Check that "Hello", $greeter->greet(null).
    }

    /** @test */
    function shouldIgnoreWhitespace() {
        $greeter = new Greeter();
        // TODO Check that "Hello Peter", $greeter->greet(" Peter ").
    }

}

File: java-junit5/src/test/java/org/codecop/Session1_GreeterTest.java
package org.codecop;

import static org.junit.jupiter.api.Assertions.assertEquals;

import org.junit.jupiter.api.DisplayName;
import org.junit.jupiter.api.Test;

/**
 * Session 1: GreeterTest - Your first tests.
 */
class Session1_GreeterTest {

    // TODO We will add the proper assertions together.
    
    @Test
    void shouldReturnHelloName() {
        Greeter greeter = new Greeter();
        // TODO Check that "Hello Peter", greeter.greet("Peter").
    }

    @Test
    @DisplayName("should return 'Hello' for 'null'")
    void shouldReturnHelloForNull() {
        Greeter greeter = new Greeter();
        // TODO Check that "Hello", greeter.greet(null).
    }

    @Test
    void shouldIgnoreWhitespace() {
        Greeter greeter = new Greeter();
        // TODO Check that "Hello Peter", greeter.greet(" Peter ").
    }
}
